src/v2/Apps/Collect/Routes/Collection/index.tsx

```diff
@@ -219,9 +219,7 @@ export const CollectionRefetchContainer = createRefetchContainer(
           ...CollectionsHubRails_linkedCollections
         }
         fallbackHeaderImage: artworksConnection(
-          aggregations: $aggregations
```

Comment:
fallback header image...does not require costly aggregations 😅

Reply:
Oh, wow. Yeah. 😬